lib/food_order.rb

```diff
@@ -0,0 +1,30 @@
+require_relative './takeaway_menu'
+
+class FoodOrder
+
+  attr_reader :food_order
```

Comment:
It might be good to rename the attribute to something else than `food_order`, to avoid mixing up with the class name